app/views/evaluation_forms/_form.html.erb

```diff
@@ -19,57 +8,93 @@
+              <option value="<%= "#{challenge.id}.1.#{challenge.end_date.strftime("%m/%d/%Y")}" %>"><%= challenge.title %></option>
+            <% end %>
+          <% end %>
+        </select>
       </div>
+      
+      <%= inline_error(evaluation_form, :challenge_id) %>
+      <%= inline_error(evaluation_form, :challenge_phase) %>
 
-      <div>
-        <%= form.label :instructions, style: "display: block" %>
-        <%= form.text_field :instructions %>
+      <%= form.hidden_field :challenge_phase, {data: { "evaluation-form-target": 'challengePhase'}} %>
+      <%= form.hidden_field :challenge_id, {data: { "evaluation-form-target": 'challengeID'}} %>
+
+      <div class="usa-character-count">
+        <div class="usa-form-group">
+          <%= form.label :instructions, class: "usa-label" %>
+          <%= form.text_area :instructions, class: "usa-textarea usa-character-count__field", maxlength: 3000, rows: "7", data: {"action": "evaluation-form#validatePresence focusout->evaluation-form#validatePresence"} %>
+        </div>
+        <span id="with-hint-textarea-info" class="usa-character-count__message">You can enter up to 3000 characters</span>
       </div>
+      <%= inline_error(evaluation_form, :instructions) %>
+
     </li>
     <li class="usa-process-list__item">
       <h4 class="usa-process-list__heading">Evaluation Criteria</h4>
       <p>
         Create your evaluation criteria and scoring options.
       </p>
-      <div>
-        <%= form.label :comments_required, style: "display: block" %>
-        <%= form.check_box :comments_required %>
-      </div>
-
-      <div>
-        <%= form.label :weighted_scoring, style: "display: block" %>
-        <%= form.check_box :weighted_scoring %>
-      </div>
     </li>
     <li class="usa-process-list__item">
       <h4 class="usa-process-list__heading">Evaluation Period</h4>
       <p>
         Set the end date of your evaluation period.
         Your evaluation start date is the end date of your challenge.
       </p>
-      <div>
-        <%= form.label :closing_date, style: "display: block" %>
-        <%= form.date_field :closing_date %>
+      <label>Evaluation Start Date:</label>
+      <div class="usa-hint" data-evaluation-form-target="startDate">mm/dd/yyyy</div>
+      <div class="usa-form-group">
+        <label class="usa-label" id="evaluation_form_closing_date_label" for="evaluation_form_closing_date"
+          >Evaluation End Date:</label
+        >
+        <div class="usa-hint" id="evaluation_form_closing_date_hint">mm/dd/yyyy</div>
+        <div class="usa-date-picker">
+          <input
+            class="usa-input"
+            id="evaluation_form_closing_date"
+            name="evaluation_form[closing_date]"
+            aria-labelledby="evaluation_form_closing_date_label"
+            aria-describedby="evaluation_form_closing_date_hint"
+            data-action="evaluation-form#validatePresence focusout->evaluation-form#validatePresence"
+          />
```

Comment:
I'm seeing a few issues here:
1. I selected Oct 1, 2025, but the date saved is "2025-01-10" which I assume is wrong. also I can't choose dates after the 12th of the month, I guess because it thinks that is the month, kind of odd.
2. When I save and edit the form again, the date is not shown in the field, it's empty. you might be able to use the [uswds property](https://designsystem.digital.gov/components/date-picker/#using-the-date-picker-component-2) `data-default-value` for this.
3. I can choose a date that is before the Evaluation Start Date. this should be validated and USWDS also provides a min date property for the date picker `data-min-date` that restricts the date picker to dates after that. could be helpful.

Reply:
Awesome, super helpful, thanks a ton